Rules/AvoidGlobalAliases.cs

```diff
@@ -0,0 +1,195 @@
+﻿using Microsoft.Windows.PowerShell.ScriptAnalyzer.Generic;
+using System;
+using System.Collections.Generic;
+#if !CORECLR
+using System.ComponentModel.Composition;
+#endif
+using System.Globalization;
+using System.Management.Automation.Language;
+
+namespace Microsoft.Windows.PowerShell.ScriptAnalyzer.BuiltinRules
+{
+#if !CORECLR
+    [Export(typeof(IScriptRule))]
+#endif
+    class AvoidGlobalAliases : AstVisitor, IScriptRule
+    {
+        private List<DiagnosticRecord> records;
+        private string fileName;
+
+        public IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string fileName)
```

Comment:
Use xml documentation comments for public methods (https://msdn.microsoft.com/en-us/library/b2s063f7.aspx)